CHANGELOG.md

```diff
@@ -1,6 +1,7 @@
 ## 2.0.28
 - Made changes to override AMQP Basic properties.
 - Swagger don't display default status codes
+- Made changes to create a new RabbitMQ Channel when it is closed in the middle of publishing.

```

Comment:
Just reading what you state hear, it sounds like there's a possibility to end up in an infinite create-channel loop (if major network issues)! What's your thoughts?